ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -67,6 +69,8 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(Cli.class);
 
+  static final int COMMAND_QUEUE_CATCHUP_TIMEOUT_RETRIES = 1;
```

Comment:
Isn't there a timeout on the server side? i.e. doesn't the server wait for some configurable amount of time before failing?   If that's the case, is there any point adding a retry?  If you want one retry, just double the timeout - same affect.

If this was configurable within the client, then I could see it might have use. But hard coded... I'd just remove it.